<launch>
  <param name="global_frame_id" value="/base_link" />	

<node pkg="cloud_tools" type="table_object_detector_sr_node" name="table_object_detector_sr_node" output="screen" respawn="true">
  <param  name="input_cloud_topic" value="/swissranger_test/cloud_sr" />;
</node>

</launch>
